### DISCUSSION TITLE
Getting Hacked Github Support No Response

### DISCUSSION THREAD

**Discussion by tintinwp** [community/community#123436] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My GitHub account got **hacked**, and it's commenting on other people's issues with a coin scam. Not long after, I deleted all my token applications and reset my password. I also deleted all the comments relevant to the coin scam one by one. However, when I finished deleting the comments, I got suspended from GitHub. After that, I couldn't access any of my repositories, including public ones.

Someone who reported me sent me an email **apologizing**, saying they mistook me for a scammer.

I've already opened a GitHub ticket about this, but it's been over **two weeks** and I haven't received any response. Is there anything I can do? _All the projects I've coded over the years are on that account, and I can't access them anymore_ 😞

PS. My Old Account : [https://github.com/TinTinWinata](url)

**Comment by IgorD-lab** [comment]
They probably just overlooked your email just send another one. 
You can also try:
https://support.github.com/contact/reinstatement

**Reply by tintinwp** [reply]
I already did it before sending an email to github, but thanks for the response 👍

**Comment by ebndev** [comment]
Hi @TinTin6892,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
